src/Draco.Debugger/Platform/IPlatformMethods.cs

```diff
@@ -8,9 +8,13 @@ namespace Draco.Debugger.Platform;
 internal interface IPlatformMethods
 {
     /// <summary>
-    /// Replaces the standard IO handles for this process.
+    /// Gets the standard IO handles of the current process.
     /// </summary>
-    /// <param name="newHandles">The new handles to be used by the process.</param>
-    /// <returns>The old IO handles.</returns>
-    public IoHandles ReplaceStdioHandles(IoHandles newHandles);
+    /// <returns></returns>
+    public IoHandles GetStdioHandles();
```

Comment:
Get is not a nice name, more like Fork or Copy?

Reply:
The behavior is platform dependent, it doesn't copy on Windows.